napari/_qt/qt_main_window.py

```diff
@@ -336,6 +350,19 @@ def _add_help_menu(self):
         )
         self.help_menu.addAction(about_key_bindings)
 
+    def _toggle_axes(self, action):
```

Comment:
I don't know where, but it should be documented *somewhere* what the default values are, for both colours and dashes. Eg: x-axis is green, etc. or whatever it really is.

Reply:
this has now been improved that the enum has been dropped